### ISSUE TITLE
deleting files/refactoring causes ng serve error

### ISSUE THREAD

**Issue by dereklin** [angular/angular-cli#1389]
I am using Windows 10.

I was doing refactoring and deleted the components folder.

But when I run ng serve again, it was still looking at the removed file in the tmp folder.  Deleting the tmp folder didn't help:

```
  c:/Users/Derek/sandbox/angular-cli/learn2/angular-cli-example-app/tmp/broccoli_type_script_compiler-input_base_path-YarT3zLE.tmp/0/src/app/components/pomodoro-tasks/pomodoro-tasks.ts (10, 22): Cannot find module '../../interfaces/task.interface'.
```

**Comment by web-dave** [comment]
could you post your System.config.ts, 
maybe forgot to clear the refs? 
`const barrels: string[] = [
...
]`

**Comment by filipesilva** [comment]
Closed as this issue was made obsolete by #1455.
